### PR TITLE
Add metrics to v2

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="sanpy",
-    version="0.7.15",
+    version="0.7.17",
```
Comment (IvanIvanoff) [inline setup.py]: why 2 versions?

Comment (spiderjako) [reply setup.py]: I was looking at the PR and the version was 0.7.16, it seems that it is actually 0.7.17 at the moment. I bumped it to 0.7.18 now

File: on-chain-metrics.md [hunk 2]
```diff
@@ -15,6 +15,11 @@ List of with metrics with 1 day minimal resolution:
 - daily_opening_price_usd
 - daily_trading_volume_usd
 - daily_active_addresses
+- active_deposits
+- active_withdrawals
+- withdrawal_transactions
+- exchange_token_supply
```
Comment (IvanIvanoff) [inline on-chain-metrics.md]: No such metric is present already

File: san/tests/test_sanbase_integration.py [hunk 3]
```diff
@@ -541,7 +534,15 @@ def test_available_metrics():
...
-        'nvt'
+        'nvt',
+        'nvt_transaction_volume',
+        'network_growth',
+        'mean_dollar_invested_age',
+        'active_deposits',
+        'active_withdrawals',
+        'withdrawal_transactions',
+        'exchange_token_supply',
+        'percent_of_total_supply_on_exchanges'
```
Comment (IvanIvanoff) [inline san/tests/test_sanbase_integration.py]: amount_in_top_holders is also present now

**Pull request merged**